connect/tests/src/test/java/com/telenor/connect/utils/InternetTimeTest.java

```diff
@@ -0,0 +1,29 @@
+package com.telenor.connect.utils;
+
+import org.junit.Test;
+
+import java.util.Calendar;
+import java.util.Date;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class InternetTimeTest {
+
+    @Test
+    public void internetTimeIsWithinBoundaryOfLocalTime() throws Exception {
+        final Date internetDate = InternetTime.getInternetDate();
+        final int boundarySeconds = 5;
+
+        Calendar calendar = Calendar.getInstance();
+        calendar.add(Calendar.SECOND, boundarySeconds);
+        Date upperLimit = calendar.getTime();
```

Comment:
Consider adding final here and for lowerLimit and withinLimit, in order to be consistent about usage of the final keyword.